.gitattributes

```diff
@@ -12,6 +12,7 @@
 # Set these files to XML (coloring) in PRs
 *.vsmanproj linguist-language=XML
 *.swixproj linguist-language=XML
+*.config linguist-language=XML
```

Comment:
I added this so it would color the `nuget.config` file in the PR. However, it doesn't seem to be working... odd. 🤔

Reply:
is this working for other extensions like  A.swixproj ?

Reply:
It is. See: https://github.com/dotnet/project-system/pull/8285/files#diff-1c7cad0a70f92aa5eb19332a8fd5fd7f973bd080fa1569265c7eed4eefa47a26

Reply:
There might be something special about `.config` I'm not aware of. For now, I'll keep this there. I'll likely need to edit this file again in the future for CFS. I'll investigate at that time.